Angular - Frontend/src/app/Services/upload-file.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpEvent, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UploadFileService {
+
+  constructor(private https: HttpClient) { }
+
+  pushFileToStorage(file: File): Observable<HttpEvent<{}>> {
+    const data: FormData = new FormData();
+
+    data.append('file', file);
+
+    const newRequest = new HttpRequest('POST', 'http://localhost:8081/uploadFile', data, {
```

Comment:
this URL should be placed in enviroment.ts

Reply:
done